week-03-reflections/lceballos-stat6863.md

```diff
@@ -3,7 +3,65 @@
 
 
 
-[place your questions/answers here, and delete this line]
+[Chapter 10, Problem 7]
+
+
+
+* Question (lceballos-stat6863): Instead of using SYMPUT, can you use %LET instead?
+
+
+
+[Chapter 10, Problem 1]
```

Comment:
Incorrect chapter numbers; please update and DM me in Slack with PR URL for re-evaluation